extensions/kafka-client/runtime/pom.xml

```
@@ -34,6 +34,11 @@
             <optional>true</optional>
         </dependency>
 
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-caffeine-deployment</artifactId>
```

```suggestion
            <artifactId>quarkus-caffeine</artifactId>
```

We should depend on the runtime module and not the `deployment` one.

---

Thanks for the suggestion. I fixed it, however I needed to add a dependency on this deployment module in `kafka-client-deployment` to make it work properly.

---

Thanks @dejanb. The CI failure seemed unrelated to change, I've re-launched the build.